src/main/java/org/mineacademy/template/command/SampleCommand.java

```diff
@@ -96,24 +98,28 @@ protected void onCommand() {
 			this.checkArgs(2, "Please specify a player's name: /{label} {sublabel} <playerName>");
 
 			// This will automatically run the method inside if the offline player was found (even if he is offline)
-			// And if not, we automatically return with an error message, editable in localization/messages_en.yml
+			// And if not, we automatically return with an error message, editable in Localization (Player.Not_Played_Before)
 			this.findOfflinePlayer(this.args[1], target -> {
 				this.tell(target.getName() + " last joined " + TimeUtil.getFormattedDateShort(target.getLastPlayed()));
 			});
 
 		} else if ("actionbar".equals(param)) {
 			this.checkArgs(3, "Usage is: /{label} {sublabel} <playerName> [message]");
 
-			// Automatically finds an online player (BungeeCord is NOT supported here)
-			// If the player is not on this server, we automatically return with an error message,
-			// editable in localization/messages_en.yml
+			// Automatically finds an online player, if the player is not on the server, 
+			// we automatically return with an error message, editable in Localization (Player.Not_Online)
 			final Player target = this.findPlayer(this.args[1]);
-			final String message = Common.joinRange(2, this.args);
+			
+			// Get the entire message starting from the second argument.
+			// /sample actionbar <player> This is the message sent into actionbar
+			final String message = this.joinRange(2);
```

Comment:
@AdoNikeOFFICE testoval si to vôbec? Taká metóda neexistuje, musel som to opraviť ručne, nabudúce to otestuj:  https://github.com/kangarko/PluginTemplate/commit/5bf1195400aaeef791715ce6d9b3c5e17b880f90